config/theme.yml

```diff
@@ -121,6 +121,43 @@ global_settings:
       width: 170
       height: 115
       scope: [stores]
+    square_xs:
```

Comment:
Since you can still change it to not be a square, then I would stick with `default` or `thumb` 😉